service/src/main/java/com/fecklessweasel/service/objectmodel/Department.java

```diff
@@ -0,0 +1,69 @@
+package com.fecklessweasel.service.objectmodel;
+
+import java.sql.Connection;
+
+import com.fecklessweasel.service.datatier.DepartmentTable;
+/**
+ * Stores all information about a school's department.
+ * @author Elliot Essman
+ */
+
+public class Department {
+	
+	private long id;
+	private int univId;
+	private String acronym;
+	private String deptName;
+	
+	private static int DEPTNAME_MAX = 20;
```

Comment:
Might consider making DEPT. name longer. For example, EECS at case is Electrical engineering and computer science and is 50ish chars.